recipes/zeromq/zeromq.inc

```diff
@@ -0,0 +1,18 @@
+inherit autotools pkgconfig c c++ auto-package-utils
+
+DESCRIPTION = "Zeromq - The Intelligent Transport Layer"
+HOMEPAGE = "http://www.zeromq.org"
+LICENSE = "LGPLv3+"
+SRC_URI = "http://download.zeromq.org/zeromq-${PV}.tar.gz"
+
+AUTO_PACKAGE_UTILS = "libzmq curve_keygen"
```

Comment:
You should remove "libzmq" here, inherit from "auto-package-libs" also, and set AUTO_PACKAGE_LIBS to "zmq", so you get proper library packaging of libzmq.